lxqt-config/lxqt-config-appearance/lxqt-config-appearance_fr.ts

```diff
@@ -0,0 +1,230 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="fr_FR">
```

Comment:
I just looked on 'github paiiou' the corrected branch and read 8101e77
Can this information help?